test/resources/Addresses.spec.js

```diff
@@ -18,6 +18,7 @@ var addressData = {
   last_name: 'Foster',
   locality: 'Chicago',
   postal_code: '2904',
+  street_address: '311 W Superior Street'
```

Comment:
Looks like you'll want to either branch from https://github.com/team-telnyx/telnyx-node/pull/49 or omit the commit from this PR, or else you'll run into merge conflicts

Reply:
I will remove from this PR and wait for the #48

Reply:
Done